test_cases/exact_matches.json

```diff
@@ -97,9 +97,10 @@
     },
     {
       "id": 8,
-      "status": "pass",
+      "status": "fail",
       "user": "sevko",
```

Comment:
add issue property to test and link to https://github.com/pelias/pelias/issues/427